recipes/pyhamcrest/meta.yaml

```diff
@@ -0,0 +1,48 @@
+package:
+  name: pyhamcrest
+  version: "1.8.3"
```

Comment:
You can use the same jinja style for the version from above here. It as the advantage to make it easy to update a recipe later on.